contracts/external/proxies/PayableProxyController.sol

```diff
@@ -70,7 +71,7 @@ contract PayableProxyController is ReentrancyGuard {
             require(sendEthTo != address(0), "PayableProxyController: cannot send ETH to address zero");
 
             weth.withdraw(remainingWeth);
-            sendEthTo.transfer(remainingWeth);
+            Address.sendValue(sendEthTo, remainingWeth);
```

Comment:
I think will be better to add `using Address for address` in the beginning, and then make the call through `sendEthTo.sendValue(remainingWeth)`